### DISCUSSION TITLE
Setting cell background color

### DISCUSSION THREAD

**Discussion by thefireescaper** [rappasoft/laravel-livewire-tables#975] [Q&A] [answered]
hi
I would like to be able to set the background color of a cell. 
Is this feasible with v2?

Thanks

**Comment by lrljoe** [comment, marked as answer]
What's the criteria for setting the background colour?

Please see the below, which would set the background of a field "total" to red if the value was less than "< 1000"

If this doesn't make sense, then please give a use case and I'll try to help give you a pointer.  The docs here:

[https://rappasoft.com/docs/laravel-livewire-tables/v2/datatable/available-methods
](https://rappasoft.com/docs/laravel-livewire-tables/v2/datatable/available-methods)

Really do cover a lot of it though!

```
public function configure(): void
{
  // Takes a callback that gives you the current column, row, column index, and row index
  $this->setTdAttributes(function(Column $column, $row, $columnIndex, $rowIndex) {
    if ($column->isField('total') && $row->total < 1000) {
      return [
        'class' => 'bg-red-500 text-white',
      ];
    }

    return [];
  });
}

```
